vue-frontend/src/components/portfolio/VideoSection.vue

```diff
@@ -196,4 +196,19 @@ section.video {
 .swiper-slide-next {
   transform: scale(0.5) !important;
 }
+
+.swiper-wrapper {
+  display: flex;
+  align-items: center;
+}
+
+.mobile-image {
+  width: 18.5%;
+}
+
+@media (min-width: 3840px) {
```

Comment:
Instead of scale, shouldn't we update width?

Reply:
in Some screen resolution images is override others so that's why i update  scale value